mvvm-state/src/commonMain/kotlin/dev/icerock/moko/mvvm/StateExt.kt

```diff
@@ -11,7 +11,7 @@ fun <T, E> T?.asState(whenNull: () -> ResourceState<T, E>): ResourceState<T, E>
     this?.asState() ?: whenNull()
 
 fun <T, E> List<T>.asState(): ResourceState<List<T>, E> = if (this.isEmpty()) {
-    ResourceState.Empty()
+    ResourceState.Empty
```

Comment:
we should also add into release notes guide to migration. please add migration regexp to pr description, we will add it into release notes later